README.md

```diff
@@ -14,63 +14,155 @@ If you wish to deploy a local explorer, run:
 $ npm run explore-local
 ```
 
+# Installation
+
+Install dependencies using `yarn`
+
+```bash
+$ yarn
+```
+
+This will start a simulated bench with the contract deployed, one `FairSale` sale and one bid.
+
 # Architecture
 
 **Fixed Contracts**
 
 The subgraph listens for events from three fixed contracts; they are deployed once:
 
 1. `MesaFactory`
-2. `AuctionLauncher`
+2. `SaleLauncher`
 3. `MesaFactory`
 
 **Dynamic Contracts**
 
-These contracts are deployed from the factory for every new auction. Their address is resolved from `Event.address` and compared against `MesaFactory.allAuctions` array. If the address does not belong to Mesa's auction, the handler does an early exit:
+These contracts are deployed from the factory for every new sale. Their address is resolved from `Event.address` and compared against `MesaFactory.allSales` array. If the address does not belong to Mesa's sale, the handler exits:
 
-1. `EasyAuction`
-2. `FixedPriceAuction`
+1. `FairSale`
+2. `FixedPriceSale`
 
 ## Contracts
 
 ### `MesaFactory`
 
 <sub>[`src/mappings/factory.ts`](src/mappings/factory.ts)</sub>
 
-#### 1. `TemplateLaunched`
-
-Emitted when `MesaFactory.launchTemplate` is called. Returns:
+#### 1. `FactoryInitialized` from `MesaFactory.Initializ`
```

Comment:
**Initialize** Is this a typo or from code? Or both. Change name in the code if a typo in a var.

Reply:
Typo